guide_to_writing_techreports.tex

```diff
@@ -212,32 +214,30 @@ \subsection{Math}
 
 Notes:
 \begin{itemize}
-\item Scalar variables ($h$, $t$, $s$) are \emph{italicised} lower case.
+\item Scalar variables ($h$, $t$, $s$) are \emph{italicised} lower case (this includes parameters).
 \item Vector variables ($\mathbf{v}$, $\mathbf{x}$) are \textbf{bold}
   lower case, but not italicised.
 \item Matrix variables ($G$) are upper case and \emph{italicised}.
   Matrices are shown in Equation~\ref{eq:matexample}, with
   square brackets.
 \item Standard mathematical functions ($\sin(\alpha)$, $\log b$), operators (d,
-  $+$, $-$) and
-  numeric constants (\num{123.4}) are not italicised.
+  $+$, $-$), numeric constants (\num{123.4}), and dimensionless numbers (Re, Nu) are set in roman type (not italicised).
 \item Numbered equations are centred, with the number in parentheses on
   right margin.
 \item A font similar to the body text font should be used, so when
   using a sans-serif font like Calibri, the formulae should be
   sans-serif as well.
 \item Differential equations are stated using the correct partial
   derivative symbol $\partial$ (not a lowercase delta $\delta$) for
-  partial derivatives and upright ``d''s  for total derivatives.
-\item Integrals are stated using upright ``d''s
+  partial derivatives and upright ``d''s  for total derivatives. Integrals are also stated using upright ``d''s.
 \item Deeply nested equations can use braces, brackets, and parentheses in the
   sequence \{[()]\}.
 \item An equation can be seen as a clause and should scan as a part of
   the sentence.  However, associated punctuation should be implied
   rather than expressed.  In Equation~\ref{eq:commaexample}, there is
   an implied comma after the equation, while in
-  Equation~\ref{eq:matexample} there is an implied period.
-\item Note the difference between the mathematical operators for subtraction $-$ and multiplication $\times$ and the en dash - and the letter x. 
+  Equation~\ref{eq:matexample} there is an implied period. However, if a citation precedes an equation, a colon may be used.
```

Comment:
This is confusing - the issue is the idea of using punctuation at the _end of an equation_, then we say you can use a colon? Isn't that about _before_ the equation?

Reply:
I was talking about the clause preceding the equation, as in this example of the definition of the mole. Will revise to be clearer
![SI_p_134_colon](https://user-images.githubusercontent.com/29544118/74637660-0d08f080-5173-11ea-928b-372d1d7e364f.png)

Reply:
This particular guidance is about the exact situation in your example. Notice that the punctuation in the equations here is expressed rather than implied (the opposite of what our guidelines suggest). The whole bullet point is about the question of putting periods or commas at the end of an equation like they have at the end of the definition of the mole. The issue of having a colon in the paragraph before the equation may merit another bullet point, but is in my mind a very different thing.